packages/flutter/test/material/bottom_app_bar_test.dart

```diff
@@ -11,7 +11,88 @@ import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+import '../rendering/mock_canvas.dart';
+
 void main() {
+  // Regression test for https://github.com/flutter/flutter/issues/123283
+  testWidgets('throws if `shape` requested in Material 3', (WidgetTester tester) async {
```

Comment:
nit:
```suggestion
  testWidgets('throws if `shape` requested in Material 3', (WidgetTester tester) async {
    // Regression test for https://github.com/flutter/flutter/issues/123283
```

and similarly below